taipy/common/config/__init__.py

```diff
@@ -126,6 +128,10 @@ def _inject_section(
     for exposed_configuration_method, configuration_method in configuration_methods:
         setattr(Config, exposed_configuration_method, configuration_method)
 
+    for type_to_register in section_clazz._types_to_register():
+        if type_to_register._type_identifier() not in _BaseSerializer._SERIALIZABLE_TYPES:
+            _BaseSerializer._SERIALIZABLE_TYPES.append(type_to_register._type_identifier())
```

Comment:
IS it necessary? I believe the `Config._register_default(default)` does the job already. no?

Reply:
I don't see anything in the `_register_default()` method that does this.

Anyway, removing this would make the serialization tests fail, so I believe it's necessary

Reply:
Oh, I understand now. Indeed, the `Config._register_default` calls the `_BaseSerializer._register` that registers the types, but does not update the `_BaseSerializer._SERIALIZABLE_TYPES` variable but the `_registered_types`.

I believe we can update the variable in the `_BaseSerializer._register` method.